csvcubed/csvcubed/utils/validators/uri.py

```diff
@@ -10,7 +10,7 @@
 from ..uri import ensure_looks_like_uri
 
 
-def validate_uri(attr_name: str, is_optional: bool = False) -> classmethod:
+def validate_uri(attr_name: list[str], is_optional: bool = False) -> classmethod:
```

Comment:
Instead of changing this functionality which is likely used elsewhere, it would be a better idea to add a new validator function and leave the existing one as-is.